backend/src/main/java/fk/prof/backend/http/LeaderHttpVerticle.java

```diff
@@ -47,11 +50,17 @@ private Router setupRouting() {
     router.post(ApiPathConstants.LEADER_POST_LOAD)
         .handler(this::handlePostLoad);
 
-    router.post(ApiPathConstants.LEADER_PUT_ASSOCIATION)
+    router.put(ApiPathConstants.LEADER_PUT_ASSOCIATION)
```

Comment:
Fixed in https://github.com/Flipkart/fk-prof/commit/a5eb03869082b66fde0437b7c51549b359204631